xbmc/URL.cpp

```diff
@@ -412,6 +412,11 @@ void CURL::SetProtocolOptions(const std::string& strOptions)
   }
 }
 
+void CURL::SetPayload(const std::string& strPayload)
+{
+  m_strPayload = strPayload;
+}
```

Comment:
Is there no other way we can do this? I'm not sure I like having this here. It's not something that belongs to the URL itself(!) does it?